xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -307,7 +307,7 @@ bool CSystemGUIInfo::GetLabel(std::string& value, const CFileItem *item, int con
       return true;
     case SYSTEM_ADDON_UPDATE_COUNT:
       value =
-          StringUtils::Format("{0}", CServiceBroker::GetAddonMgr().GetAvailableUpdates().size());
+          StringUtils::Format("{0}", CServiceBroker::GetAddonMgr().GetLastAvailableUpdatesCount());
```

Comment:
If the new method is made exactly for this use case, i would let it do the string conversion, which makes it even cheaper. Means, change the new member to be a string.